futures-util/src/try_stream/try_for_each.rs

```diff
@@ -0,0 +1,63 @@
+use core::marker::Unpin;
+use core::mem::PinMut;
+use futures_core::future::{Future, TryFuture};
+use futures_core::stream::TryStream;
+use futures_core::task::{self, Poll};
+
+/// A stream combinator which attempts to execute an async closure over each
+/// future in a stream.
+///
+/// This structure is returned by the `TryStream::try_for_each` method.
```

Comment:
This future is returned by the `TryStream::try_for_each` method.

Reply:
Done.